curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-067.md

```diff
@@ -0,0 +1,689 @@
+---
+id: 6148e5aeb102e3142de026a2
+title: Part 67
+challengeType: 0
+dashedName: part-67
+---
+
+# --description--
+
+Finally, the navigation accessibility can be improved with providing keyboard shortcuts.
+
+The `accesskey` attribute accepts a space-separated list of access keys. For example:
+
+```html
+<button type="submit" access-key="s">Submit</button>
+```
+
+Give each of the navigation links a single-letter access key.
+
+_Note: It is not always advised to use access keys, but they can be useful_
+
+Well done. You have completed the _Accessibility Quiz_ practice project.
+
+# --hints--
+
+You should give the first `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[0]?.accessKey?.length, 1);
+```
+
+You should give the second `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[1]?.accessKey?.length, 1);
+```
+
+You should give the third `a` element a single-letter `accesskey`.
+
+```js
+assert.equal(document.querySelectorAll('a')?.[2]?.accessKey?.length, 1);
+```
```

Comment:
```suggestion
You should give the first `a` element a single-letter `accesskey`.

```js
assert.equal(document.querySelectorAll('a')?.[0]?.getAttribute('access-key')?.length, 1);
```

You should give the second `a` element a single-letter `accesskey`.

```js
assert.equal(document.querySelectorAll('a')?.[1]?.getAttribute('access-key')?.length, 1);
```

You should give the third `a` element a single-letter `accesskey`.

```js
assert.equal(document.querySelectorAll('a')?.[2]?.getAttribute('access-key')?.length, 1);
```
```

Reply:
This specifically did not work for me 😖 😕